modules/api-compatibility-common-terminology.adoc

```diff
@@ -0,0 +1,52 @@
+
+[id="api-compatibility-common-terminology-cloud_{context}"]
+== Compatibility in a cloud environment
+
+{product-title} might choose to offer integration points with a hosting cloud environment via cloud provider specific integrations. The compatibility of these integration points are specific to the guarantee provided by the native cloud vendor and its intersection with the {product-title} compatibility window.  Where {product-title} provides an integration with a cloud environment natively as part of the default installation, Red Hat develops against stable cloud API endpoints to provide commercially reasonable support with forward looking compatibility that includes stable deprecation policies. Example areas of integration between the cloud provider and {product-title} include but are not limited to dynamic volume provisioning, service load balancer integration, pod workload identity, dynamic management of compute, and infrastructure provisioned as part of initial installation.
+
+[id="api-compatibility-common-terminology-releases_{context}"]
+== Major, minor, and z-stream releases
+
+A Red Hat major release represents a significant step in the development of a product. Minor releases appear more frequently within the scope of a major release and represent deprecation boundaries that might impact future application compatibility. A z-stream release is an update to a minor release which provides a stream of continuous fixes to an associated minor release. API and AOE compatibility is never broken in a z-stream release except when this policy is explicitly overridden in order to respond to an unforeseen security impact.
+
+For example, in the release 4.3.2:
+
+* 4 is the major release version
+* 3 is the minor release version
+* 2 is the z-stream release version
+
+[id="api-compatibility-common-terminology-eus_{context}"]
+== Extended user support (EUS)
+
+A minor release in an {product-title} major release that has an extended support window for critical bug fixes. Users are able to migrate between EUS releases by incrementally adopting minor versions between EUS releases. It is important to note that the deprecation policy is defined across minor releases and not EUS releases. As a result, an EUS user might have to respond to a deprecation when migrating to a future EUS while sequentially upgrading through each minor release.
+
+[id="api-compatibility-common-terminology-dev-preview_{context}"]
+== Developer Preview
+
+An optional product capability that is not officially supported by Red Hat, but is intended to provide a mechanism to explore early phase technology. By default, Developer Preview functionality is opt-in, and subject to removal at any time. Enabling a Developer Preview feature might render a cluster unsupportable dependent upon the scope of the feature.
+
+[id="api-compatibility-common-terminology-tech-preview_{context}"]
+== Technology Preview
```

Comment:
it's not clear to me what the effective difference is between tech preview and dev preview.  is there one?

Reply:
tech preview has a higher commitment to progression to GA.

dev preview items are not in product doc.
tech preview items are captured in release notes: https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-deprecated-removed-features